common/src/sql/dbinit.sql

```diff
@@ -34,6 +34,12 @@ CREATE DATABASE omicron;
 CREATE USER omicron;
 ALTER DEFAULT PRIVILEGES GRANT INSERT, SELECT, UPDATE, DELETE ON TABLES to omicron;
 
+/*
+ * Configure a replication factor of 5 to ensure that the system can maintain
+ * availability in the face of any two node failures.
+ */
+ALTER RANGE default CONFIGURE ZONE USING num_replicas = 5;
```

Comment:
For local testing CRDB -- e.g. with the omicron-dev tool -- what does this do?

Reply:
Good question.  I expect it will have no practical impact.  The default replication factor is 3, which is already more nodes than we ever have today.  The impact is that CockroachDB considers all of its data under-replicated.  It reports it as such (e.g., [here](https://www.cockroachlabs.com/docs/stable/query-replication-reports.html#find-out-which-databases-and-tables-have-under-replicated-ranges)) and if new nodes show up, it will try to replicate data until nothing is under-replicated.

One could imagine that this would instead disallow you from writing any data to the cluster when it can't satisfy this constraint.  That doesn't seem to be how it works.  If it did, the tests wouldn't run at all, I think.